src/Adyen/Service/ResourceModel/BinLookup/GetCostEstimate.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Adyen\Service\ResourceModel\BinLookup;
+
+class GetCostEstimate extends \Adyen\Service\AbstractResource
+{
```

Comment:
```suggestion
{
    /**
     * @var string
     */
    protected $endpoint;
    
    
    /**
     * Include applicationInfo key in the request parameters
     *
     * @var bool
     */
    protected $allowApplicationInfo = false;
    
```

Let's do like the other services?